pkg/notary/log.go

```diff
@@ -0,0 +1,66 @@
+package notary
+
+import (
+	"fmt"
+
+	"github.com/go-logr/logr"
+	notationlog "github.com/notaryproject/notation-go/log"
+)
+
+func NotaryLoggerAdapter(logger logr.Logger) notationlog.Logger {
+	return &notaryLoggerAdapter{
+		logger: logger,
+	}
+}
+
+type notaryLoggerAdapter struct {
+	logger logr.Logger
+}
+
+func (nla *notaryLoggerAdapter) Debug(args ...interface{}) {
+	nla.logger.V(4).Info(fmt.Sprint(args...))
+}
+
+func (nla *notaryLoggerAdapter) Debugf(format string, args ...interface{}) {
+	nla.logger.V(4).Info(fmt.Sprintf(format, args...))
+}
+
+func (nla *notaryLoggerAdapter) Debugln(args ...interface{}) {
+	nla.logger.V(4).Info(fmt.Sprintln(args...))
```

Comment:
This code is duplicated for `Infoln` and `Warnln`, we should factorise this out.